misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/CollectionAssertionsSpec.kt

```diff
@@ -27,10 +36,13 @@ abstract class CollectionAssertionsSpec(
     val isDescr = DescriptionBasic.IS.getDefault()
     val isNotDescr = DescriptionBasic.IS_NOT.getDefault()
     val empty = DescriptionCollectionAssertion.EMPTY.getDefault()
+    val fluent = expect(listOf(1, 2) as Collection<Int>)
+    val sizeDescr = DescriptionCollectionAssertion.SIZE.getDefault()
 
-    describeFun(isEmpty, isNotEmpty) {
+    describeFun(isEmpty, isNotEmpty, sizeFeature, size) {
```

Comment:
please move to own describeFun. We usually group functions which take part in all tests this is not the case here, we don't make an assertion about size in context collection is empty and vice versa for isEmpty/isNonEmpty in context list with two entries.

Reply:
:+1: Will do - didn't think about that when fusing :(

Reply:
No problems